include/CAPI.h

```diff
@@ -0,0 +1,40 @@
+#pragma once
+#include "ClientConfig.h"
+#include "Client.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+   struct c_client;
```

Comment:
I would like a `c_` prefix to differentiate from C++ world. Yes, I think `voltdb` is necessary in the name in case some other DB client is linked to the executable. So I'll change it to `c_voltdb_client` and the same for the rest.